Documentation/workflow/EventPipeDeveloperGuide.md

```diff
@@ -0,0 +1,105 @@
+
+# Event Pipe Developer Guide
+
+The goal of this guide is to give a good introduction for new contributors who would like to develop Event Pipe.
+
+## What is Event Pipe?
+
+It's new, cross-platform mechanism for fast event tracing in .NET Core. [Here](https://github.com/dotnet/designs/blob/master/accepted/cross-platform-performance-monitoring.md) you can read more about its design. 
+
+The key things are:
+
+* no OS-specific dependencies
+* writer writes to a binary file, a reader reads directly from the file
+* it's not a replacement, but rather an extension
```

Comment:
Write this in the positive for clarity. E.g. "It is an extension, working in parallel or independently of the platform-specific tracing (e.g ETW, LTTng)"